doc/changelog.md

```diff
@@ -69,6 +69,18 @@
 
 <h3>Improvements</h3>
 
+* `debug.callbacks` are marked as inactive. This means `debug.callbacks` will not be considered
+  as active for the computation of gradients.
+  [(#706)](https://github.com/PennyLaneAI/catalyst/pull/706)
+
+* `callback.pure_callback` are now lowered to `ActiveCallbackOp`s.
+  Which will be treated as active callbacks by Enzyme. During the lowering of
+  `ActiveCallbackOp`s, the callback will create a specialized function for the
+  `ActiveCallback`. This is necessary for registering custom gradients with
+  Enzyme.
+  [(#735)](https://github.com/PennyLaneAI/catalyst/pull/735)
+
+
```

Comment:
Add your name to the contributors list :)

Reply:
Ok! But if you are reviewing review #706 first. :sweat_smile:

Reply:
I didn't get a notification of this message, sorry :(